web/views/manifestation.py

```diff
@@ -34,7 +34,7 @@ def post(self):
         parser = reqparse.RequestParser()
         parser.add_argument('manifestation', type=manifestation_model,
                             required=True, location='json')
-        parser.add_argument('work', type=work_model, required=True,
+        parser.add_argument('copyright_holder', type=work_model, required=True,
```

Comment:
Run into a bit of a conflict with snake case vs camel case here. Should the payload be completely in camel case?